sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -541,12 +545,18 @@ def __init__(self, message, response, parts):
 class BatchErrorException(HttpResponseError):
     """There is a failure in batch operations.
 
-    :param str message: The message of the exception.
+    :param message: The message of the exception.
+    :type message: str
     :param response: Server response to be deserialized.
-    :param list parts: A list of the parts in multipart response.
+    :type response: str
+    :param parts: A list of the parts in multipart response.
+    :type parts: List[str]
+    :param args: Args to be passed to :class:`~azure.core.exceptions.HttpResponseError`
+    :type args: List[str]
     """
 
     def __init__(self, message, response, parts, *args, **kwargs):
+        # type: (str, str, List[str], List[str], Dict[str, Any]) -> None
```

Comment:
This is inaccurate - response is a HttpResponse object, parts is a list of responses.